lisa/tools/nvmecli.py

```diff
@@ -39,16 +43,36 @@ def _install_from_src(self) -> None:
         make.make_install(cwd=code_path)
 
     def create_namespace(self, namespace: str) -> ExecutableResult:
-        return self.run(f"create-ns {namespace}", shell=True, sudo=True)
+        cmd_result = self.run(f"create-ns {namespace}", shell=True, sudo=True)
+        if self.__missing_block_size_parameter in cmd_result.stdout:
```

Comment:
1. It looks there pattern is the same. Because there are more lines now, it's time to have a common method to do it.
2. Add comments to explain the condition, why the block-size may not found.

Reply:
Double check, only create command need specify --block-size when nvme version is greater than 1.6